Orange/widgets/data/owfile.py

```diff
@@ -101,11 +101,26 @@ def __init__(self):
         button = gui.button(
             box, self, "Reload", callback=self.reload, autoDefault=False)
         button.setIcon(self.style().standardIcon(QtGui.QStyle.SP_BrowserReload))
-        button.setSizePolicy(Policy.Fixed, Policy.Fixed)
+        button.setSizePolicy(Policy.Fixed, Policy.Fixed) 
+        hBLayout = QtGui.QHBoxLayout()
+        self.sheet_combo = QtGui.QComboBox()
+        self.sheet_combo.setSizePolicy(Policy.MinimumExpanding, Policy.Fixed)
+        self.sheet_combo.setSizePolicy(Policy.MinimumExpanding, Policy.Fixed)
```

Comment:
Elephants and pythons never forget. Setting the size policy once suffices. No need to repeat. :))))